tests/child-contract-v2.test.sh

```
@@ -61,6 +62,9 @@ cmp "$ROOT/plugins/uberdev/shared/phase1-reviewer-output-v1.md" \
 # The shared constructor must enforce RepoPathArray before the later handoff
 # boundary so a successful build always carries the same nominal invariant.
 INPUT_HELPER="$ROOT/plugins/uberdev/lib/child-inputs.py"
+! grep -q 'MAX_INPUT_BYTES[[:space:]]*=' "$INPUT_HELPER"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)